### ISSUE TITLE
Backpack looks nothing like in Scratch.

### ISSUE THREAD

**Issue by Radical-from-Scratch** [NoMod-Programming/Whiskers-2.8#11]
I'm going to keep it the way it is, but it's worth looking in to.

**Comment by NoMod-Programming** [comment]
Yup. I'm not fully done with everything at this point. The backpack was mainly rushed and ported from something else I did (NoMod). But I might have a better backpack soon (and maybe it can support sounds and costumes and whatever :smile:

**Comment by Radical-from-Scratch** [comment]
Yeah, just do whatever you did to make a tips bar (even though it's empty :P)

**Comment by NoMod-Programming** [comment]
Yeah. I'm working on both a tips bar and a backpack.
